content/posts/gro-gamlestaden-skyltar-2024-08-25.md

```diff
@@ -0,0 +1,36 @@
+title: GRO Gamlestaden
+date: 2024-08-25
+modified:
+author: Anders Nord
+category: Projekt
+tags: klimatanpassning, politik, lokalt, mat, lokalproducerat, klimat, ifk
+status: published
+data:
+
+<div class="post-image-left">
+    <img alt="GRO skylt" src="data/gro_skylt.jpg" />
+    <em>Informationsskylt vid plantering i Gamlestaden. Foto: GRO.</em>
+</div>
+
+**Föreningen GRO gamlestaden har sedan tidigare fått hjälp med en IFK-administrerad
+hemsida. Sedan odlingen tagit fart har även informationsskyltar designats och producerats.
+Dessa har nu funnit sina platser vid odlingarna.**
+
+Skyltarna, som designats av arkitekten Anna Katarina Baudner med hjälp av industridesignern
+Peter Mikaelsson, trycktes och levererades före sommaren. Medlemmar i föreningen
+GRO har sedan under sommaren snickrat ramar och kompletterat skyltarna med utbytbara
+kartor över odlingslotterna.
+
+Ingenjörsarbete För Klimatet är glada att kunna hjälpa till och kan intyga att potatisen
+från odlingarna i Gamlestaden är väldigt goda!
```

Comment:
Maybe it's me, but is this sentence correct?

Reply:
You might be right - regardless, it looks a little bit cumbersome. I'm simplifying. 👍